matrix_client/client.py

```diff
@@ -100,7 +105,7 @@ def global_callback(incoming_event):
 
     def __init__(self, base_url, token=None, user_id=None,
                  valid_cert_check=True, sync_filter_limit=20,
-                 cache_level=CACHE.ALL, encryption=False):
+                 cache_level=CACHE.ALL, encryption=False, encryption_conf={}):
```

Comment:
Default args to methods should almost never be mutable objects. See #236 for example of bugs caused.